src/Infrastructure/Persistence/Migrations/20200115010414_Initial.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
```

Comment:
This file will need updating to match field name changes.

Reply:
The command I mentioned (`ef migrations add`) always adds a new migration file. We technically should never edit these files. Only remove/add

Reply:
Ah, missed that this was a migration. We'll need a new migration to match the changes, in that case.